extensions/wikia/Email/Controller/WatchedPageController.class.php

```diff
@@ -83,12 +82,10 @@ public function body() {
 			'editorUserName' => $this->getCurrentUserName(),
 			'editorAvatarURL' => $this->getCurrentAvatarURL(),
 			'details' => $this->getDetails(),
-			'buttonText' => $this->getCompareChangesLabel(),
-			'buttonLink' => $this->getCompareChangesLink(),
-			'contentFooterMessages' => [
-				$this->getArticleLinkText(),
-				$this->getAllChangesText(),
-			],
+			'buttonText' => $this->getButtonText(),
+			'buttonLink' => $this->getButtonLink(),
+			'contentFooterMessages' => $this->getContentFooterMessages(),
+			'contentFooterMessagesCount' => ( bool ) count( $this->getContentFooterMessages() ),
```

Comment:
Maybe this should be named for how is used rather than how its calculated; ```hasContentFooterMessages``` maybe?